Author: Andreas Beckmann <anbe@debian.org>
Description: enable hardening flags while building the GUI
 drop -v flag since it confuses blhc

--- a/GUI/i7z_GUI.pro
+++ b/GUI/i7z_GUI.pro
@@ -2,6 +2,10 @@
 # Automatically generated by qmake (2.01a) Tue Jan 26 22:31:53 2010
 ######################################################################
 
+QMAKE_CFLAGS	*= $(shell dpkg-buildflags --get CFLAGS)   $(shell dpkg-buildflags --get CPPFLAGS)
+QMAKE_CXXFLAGS	*= $(shell dpkg-buildflags --get CXXFLAGS) $(shell dpkg-buildflags --get CPPFLAGS)
+QMAKE_LFLAGS	*= $(shell dpkg-buildflags --get LDFLAGS)
+
 TEMPLATE = app
 TARGET = 
 DEPENDPATH += .
@@ -9,7 +13,7 @@ INCLUDEPATH += .
 CONFIG += debug
 CFLAGS +=  #QMAKE_*FLAGS are where the gcc flags should be passed
 CXXFLAGS += 
-QMAKE_CFLAGS += -v -fno-schedule-insns2 -fno-schedule-insns -fno-inline-small-functions -fno-caller-saves 
-QMAKE_CXXFLAGS += -v -fno-schedule-insns2 -fno-schedule-insns -fno-inline-small-functions -fno-caller-saves
+QMAKE_CFLAGS += -fno-schedule-insns2 -fno-schedule-insns -fno-inline-small-functions -fno-caller-saves 
+QMAKE_CXXFLAGS += -fno-schedule-insns2 -fno-schedule-insns -fno-inline-small-functions -fno-caller-saves
 # Input
 SOURCES += i7z_GUI.cpp
